java/vector/src/main/java/org/apache/arrow/vector/ipc/message/ArrowMessage.java

```diff
@@ -26,6 +26,8 @@
 
   <T> T accepts(ArrowMessageVisitor<T> visitor);
 
+  byte getType();
```

Comment:
`getType` is a little ambiguous. How about `getHeaderType`?

Reply:
renamed to something clearer and added comment.